teams/3/list-examples.html

```diff
@@ -0,0 +1,152 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <link rel="stylesheet" href="./styles.css" />
+        <link rel="stylesheet" href="../../css/normalize.css" />
+        <link rel="stylesheet" href="../../css/main.css" />
+        <link rel="stylesheet" href="../../css/vertical-navigation.css" />
+        <title>Document</title>
+    </head>
+    <body>
+        <h1>List examples</h1>
+        <section>
+            <h2>Example number 1</h2>
+            <ul class="ut-list-cols-2">
+                <li class="text-xl-med">
+                    <img src="./assets/clock.svg" alt="clock" />
+                    <h3>Access to daily analytics</h3>
+                    <p>
+                        Optimize the way you recover, train, and sleep with
+                        daily reporting on mobile and desktop apps. Start
+                        training smarter, not harder.
+                    </p>
+                </li>
+                <li class="text-xl-med">
+                    <img src="./assets//charge.svg" alt="charge" />
+                    <h3>Measure recovery</h3>
+                    <p>
+                        The most advanced sleep tracking technology available
+                        today. Measure and track your recovery to unlock your
+                        greatest potential.
+                    </p>
+                </li>
+                <li class="text-xl-med">
+                    <img src="./assets//phone.svg" alt="phone" />
+                    <h3>Tech that evolves with you</h3>
+                    <p>
+                        Know where your strengths lie and where you can improve.
+                        Untitled provides the latest tech with a steady stream
+                        of new features.
+                    </p>
+                </li>
+                <li class="text-xl-med">
+                    <img src="./assets//social.svg" alt="social" />
+                    <h3>Unrivalled community</h3>
+                    <p>
+                        Join teams in the app with friends, family, and
+                        like-minded fitness enthusiasts. Create custom teams
+                        based on activities and interests.
+                    </p>
+                </li>
+            </ul>
+        </section>
+        <section>
+            <h2>Example number 2</h2>
+            <ul class="ut-list-cols-3">
+                <li class="text-md-reg">
+                    <img src="./assets//notion.svg" alt="notion icon" />
```

Comment:
In my computer, it does not have a problem. I checked in Chrome, Safari, and Firefox. Which browser are you using? 
<img width="1397" alt="Screen Shot 2021-11-22 at 10 19 48 PM" src="https://user-images.githubusercontent.com/51405572/142936939-c9225af0-ae08-46fc-a7b8-4f908373f046.png">

Reply:
For me works now, not sure why yesterday was not. One last thing the last list example does not show the icon, seems that the icon is missing. Please check and then we can merge

![image](https://user-images.githubusercontent.com/7091284/143023056-ef3d798a-09b8-41c5-9040-68b4b5d89d2d.png)

Reply:
Funny, because I just checked again and in all three browsers it's working fine :D
<img width="388" alt="Screen Shot 2021-11-23 at 2 26 15 PM" src="https://user-images.githubusercontent.com/51405572/143032400-f56e9acb-8e57-4c60-a5e3-7a5a1615faac.png">
Could you check one more time? If it still not working, that I'll try something else.